### PR TITLE
Improving visualization in FT notebooks images

### PR DESCRIPTION
**Pull request by shubhamiit**
# Description

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by shubhamiit** [comment]
Screenshots
OD
classification
IS
multilabel

File: sdk/python/foundation-models/system/finetune/image-classification/multilabel-classification/hftransformers-fridgeobjects-multilabel-classification.ipynb [hunk 1]
```diff
@@ -964,43 +964,109 @@
...
-    "IMAGE_SIZE = (18, 12)\n",
-    "plt.figure(figsize=IMAGE_SIZE)\n",
     "img_np = mpimg.imread(sample_image)\n",
     "img = Image.fromarray(img_np.astype(\"uint8\"), \"RGB\")\n",
     "x, y = img.size\n",
-    "\n",
-    "fig, ax = plt.subplots(1, figsize=(15, 15))\n",
-    "# Display the image\n",
-    "ax.imshow(img_np)\n",
+    "score_threshold = 0.5 #display top objects with confidence score > 0.5\n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/image-classification/multilabel-classification/hftransformers-fridgeobjects-multilabel-classification.ipynb:1002]:
```suggestion
    "score_threshold = 0.5 # display top objects with confidence score > 0.5\n",
```

File: sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb [hunk 2]
```diff
@@ -956,39 +956,75 @@
     "resp"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Visualize detections\n",
+    "Now that we have scored a test image, we can visualize the bounding boxes and masks for this image"
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb:964]:
```suggestion
    "Now that we have scored a test image, we can visualize the bounding boxes and masks for this image."
```

File: sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb [hunk 3]
```diff
@@ -956,39 +956,75 @@
...
-    "import numpy as np\n",
-    "import json\n",
     "\n",
-    "IMAGE_SIZE = (18, 12)\n",
-    "plt.figure(figsize=IMAGE_SIZE)\n",
+    "\n",
     "img_np = mpimg.imread(sample_image)\n",
     "img = Image.fromarray(img_np.astype(\"uint8\"), \"RGB\")\n",
     "x, y = img.size\n",
+    "conf_threshold = 0.6 #display top objects with confidence score > 0.6 \n",
```
Comment (rjaincc) [inline sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb:987]:
```suggestion
    "conf_threshold = 0.6 # display top objects with confidence score > 0.6 \n",
```

**Review by rjaincc — APPROVED** [review]
Left some minor comments. 
Also, why keep different thresholds - 0.5 and 0.6?

**Review by gauravrajguru — APPROVED** [review]

**Comment by shubhamiit** [comment]
> Left some minor comments. Also, why keep different thresholds - 0.5 and 0.6?

Followed the same thresholds, as v2 notebooks.

**Pull request merged**